src/portable/portable.c

```diff
@@ -1674,8 +1675,20 @@ int portable_detach(
         return ret;
 
 not_found:
-        log_debug("No unit files associated with '%s' found. Image not attached?", name_or_path);
-        return sd_bus_error_setf(error, BUS_ERROR_NO_SUCH_UNIT, "No unit files associated with '%s' found. Image not attached?", name_or_path);
+        extensions = strv_join(extension_image_paths, ", ");
+        if (!extensions)
+                return -ENOMEM;
+
+        log_debug("No unit files associated with '%s%s%s' found attached to the system. Image not attached?",
+                  name_or_path,
+                  isempty(extensions) ? "" : "' or any of its extensions '",
+                  isempty(extensions) ? "" : extensions);
+        return sd_bus_error_setf(error,
+                                 BUS_ERROR_NO_SUCH_UNIT,
+                                 "No unit files associated with '%s%s%s' found attached to the system. Image not attached?",
+                                 name_or_path,
+                                 isempty(extensions) ? "" : "' or any of its extensions '",
+                                 isempty(extensions) ? "" : extensions);
```

Comment:
please reuse the message. i.e.:

```c
r = sd_bus_error_setf(error, …);
log_debug_errno(r, "%s", error.mesage);
return r;
```

i.e. no need to have the very same format string in place twice here.

Reply:
pre-existing condition :-) but sure, fixed